servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -1639,6 +1645,43 @@ private TypeSpec newServiceInterfaceSpec(final State state, final boolean blocki
                 .addStatement("return $L", blocking ? BLOCKING_METHOD_DESCRIPTORS : ASYNC_METHOD_DESCRIPTORS)
                 .build());
 
+        // generate default service methods
+        if (defaultServiceMethods) {
+            final List<MethodDescriptorProto> methodList = state.serviceProto.getMethodList();
+            for (int i = 0; i < methodList.size(); i++) {
+                final MethodDescriptorProto methodProto = methodList.get(i);
+                final ClassName inClass = messageTypesMap.get(methodProto.getInputType());
+                final ClassName outClass = messageTypesMap.get(methodProto.getOutputType());
+                final String methodName = sanitizeIdentifier(methodProto.getName(), true);
+                final String serviceName = isNullOrEmpty(state.serviceProto.getName()) ?
+                        "" : state.serviceProto.getName();
+                final String fullMethodName = serviceName + "/" + methodName;
+                final int methodIndex = i;
+                interfaceSpecBuilder.addMethod(newRpcMethodSpec(inClass, outClass, methodName,
```

Comment:
On the other hand, `BlockingTesterService` has 2 methods that should be there only when `slipDeprecated=false`

1.
```java
        @Deprecated
        default void testBiDiStream(GrpcServiceContext ctx, BlockingIterable<TestRequest> request,
                GrpcPayloadWriter<TestResponse> responseWriter) throws Exception {
            throw new GrpcStatusException(new GrpcStatus(GrpcStatusCode.UNIMPLEMENTED, "Method Tester/testBiDiStream is unimplemented"));
        }
```

2.
```java
        @Deprecated
        default void testResponseStream(GrpcServiceContext ctx, TestRequest request,
                GrpcPayloadWriter<TestResponse> responseWriter) throws Exception {
            throw new GrpcStatusException(new GrpcStatus(GrpcStatusCode.UNIMPLEMENTED, "Method Tester/testResponseStream is unimplemented"));
        }
```